scripts/local/setup.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Define variable to check from other scripts if this script is being invoked.
+export YALESITES_INSTALL=1
```

Comment:
Let's plan to either rename this something that describes the utility such as IS_LOCAL 
OR we can try another check in the content-import.sh script to see if lando is available. It would be best to programmatically check if we are on local if possible.